reports/migrations/0013_test_testinstance.py

```diff
@@ -0,0 +1,75 @@
+# Generated by Django 4.2.7 on 2024-01-17 17:56
+
+import uuid
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
```

Comment:
TODO: add a sql migrate here, command `python manage.py sqlmigrate report 0013`, see sample of end result here https://github.com/codecov/codecov-api/blob/86431d3ac1d8d982590eede727bcb8b2afe982b0/codecov_auth/migrations/0045_remove_ownerprofile_terms_agreement.py#L8